modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
 sudo chmod +x rtfctl
 ----
 
-== List Supported Commands
+== Supported Commands
 
-To list all supported commands, run `rtfctl -h`. 
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Command |Description
```

Comment:
Hmm...it always sort of bothers me when the entries in a column titled "Description" are verb-based rather than noun-based. We are not here telling the reader what to DO with the command; we are describing what the command does. I would reword each of these to "specifies," "applies," and so on.
Or I would change "Description" to imply a user action instead.